modules/internal/DefaultRectangular.chpl

```diff
@@ -21,7 +21,7 @@
 //
 module DefaultRectangular {
   config const dataParTasksPerLocale = 0;
-  config const dataParIgnoreRunningTasks = if CHPL_LOCALE_MODEL=="numa" then true
+  config const dataParIgnoreRunningTasks = if CHPL_LOCALE_MODEL!="flat" then true
```

Comment:
I've added this dataParIgnoreRunningTasks item to issue #5569 as a non-KNL-related issue that should be investigated at a later date.